backend/src/v4/models/project.js

```diff
@@ -468,10 +471,21 @@
 			project.name = data.name;
 		}
 
+		let permissionChange;
 		if (data.permissions) {
 			await Promise.all(data.permissions.map(async (permissionUpdate) => {
 				const userIndex = project.permissions.findIndex(x => x.user === permissionUpdate.user);
 
+				if(!permissionChange) {
+					permissionChange = {
+						users: [],
+						from: project.permissions[userIndex]?.permissions ?? null,
```

Comment:
so we're making the assumption here that for all permissions update, they're all had the same initial permissions and the same new permissions?

Is this a fair assumption to make?